packages/preview/fontawesome/0.4.0/README.md

```diff
@@ -0,0 +1,149 @@
+# typst-fontawesome
+
+A Typst library for Font Awesome icons through the desktop fonts.
+
+p.s. The library is based on the Font Awesome 6 desktop fonts (v6.6.0)
+
+## Usage
+
+### Install the fonts
+
+You can download the fonts from the official website: https://fontawesome.com/download
+
+After downloading the zip file, you can install the fonts depending on your OS.
+
+#### Typst web app
+
+You can simply upload the `otf` files to the web app and use them with this package.
+
+#### Mac
+
+You can double click the `otf` files to install them.
+
+#### Windows
+
+You can right-click the `otf` files and select `Install`.
+
+#### Some notes
+
+This library is tested with the otf files of the Font Awesome Free set. TrueType fonts may not work as expected. (Though I am not sure whether Font Awesome provides TrueType fonts, some issue is reported with TrueType fonts.)
+
+### Import the library
+
+#### Using the typst packages
+
+You can install the library using the typst packages:
+
+`#import "@preview/fontawesome:0.3.0": *`
```

Comment:
You probably want to update the version here :)

Reply:
Oh, my mistake.